LICENSE-MIT.txt

```diff
@@ -0,0 +1,22 @@
+MIT License
+
+Copyright (c) 2024 Hassana Labs
```

Comment:
The only reason for this file is that we want to point to the MIT license text. No need to mention Hassana Labs here.
```suggestion
```